modules/olm-updating-operator-catalog-image.adoc

```diff
@@ -0,0 +1,151 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-restricted-networks.adoc
+
+[id="olm-updating-operator-catalog-image_{context}"]
+= Updating an Operator catalog image
+
+After a cluster administrator has configured OperatorHub to use custom Operator
+catalog images, administrators can keep their {product-title} cluster up to date
+with the latest Operators by capturing updates made to Red Hat’s App Registry
+catalogs. This is done by building and pushing a new Operator catalog image,
+then replacing the existing  CatalogSource’s `spec.image` parameter with the new
+image digest.
+
+For this example, the procedure assumes a custom `redhat-operators` catalog
+image is already configured for use with OperatorHub.
+
+.Prerequisites
+
+* A Linux workstation with unrestricted network access
+ifeval::["{context}" == "olm-restricted-networks"]
+footnoteref:[BZ1771329]
+endif::[]
+* `oc` version 4.3+
+* `podman` version 1.5.1+
+* Access to mirror registry that supports
+link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2], which
+must be exposed with a certificate that is trusted by the workstation
```

Comment:
this requirement has been relaxed - you can now pass `--insecure` if you don't want to configure trust for the target registry.